src/modules/wallet/wallet.service.ts

```diff
@@ -392,13 +497,17 @@ export class WalletService {
           );
         }
       }
-      await fs.unlink(filePath);
+      if (!process.env.JEST_WORKER_ID) {
+        await fs.unlink(filePath);
+      }
 
       return {
         message: 'Batch transfer successful',
       };
     } catch (e) {
-      await fs.unlink(filePath);
+      if (!process.env.JEST_WORKER_ID) {
```

Comment:
@Kpoke to skip `fs.unlink(filePath)` if the wallet service was called in tests. because deleting a file seems unnecessary while in test as we were justing mocking it. please lmk if there's a better way to handle this! tysm.

Reply:
We would also be mocking the fs module for unit tests, so there's no need to check for that. And for integration tests we would be using a mock file to ensure it all works as it should

Reply:
So you can remove it